code/game/machinery/turrets.dm

```diff
@@ -566,4 +570,10 @@
 /obj/machinery/gun_turret/grenade
 	name = "mounted grenade launcher (40mm)"
 	desc = "Syndicate 40mm grenade launcher defense turret. If you've had this much time to look at it, you're probably already dead."
-	bullet_type = /obj/item/projectile/bullet/a40mm
\ No newline at end of file
+	bullet_type = /obj/item/projectile/bullet/a40mm
+
+/obj/machinery/gun_turret/assault_pod
+	name = "machine gun turret (4.6x30mm)"
+	desc = "Syndicate exterior defense turret chambered for 4.6x30mm rounds. Designed to be fitted to assault pods, it uses low calliber bullets to save space."
+	health = 100

```

Comment:
@Fox-McCloud and turrets have a default health of 80 on -tg-
Our syndicate-ship has turrets with 200 health. Same ratio.